www/data_service/src/services/rest/rest.service.coffee

```diff
@@ -0,0 +1,35 @@
+class Rest extends Service
+    constructor: ($http, $q, API) ->
+        return new class RestService
+            constructor: ->
+
+            execute: (config) ->
+                $q (resolve, reject) =>
+                    $http(config)
+                    .success (response) ->
+                        try
+                            data = angular.fromJson(response)
+                            resolve(data)
+                        catch e
+                            reject(e)
+                    .error (reason) -> reject(reason)
+
+            get: (url, params = {}) ->
+                config =
+                    method: 'GET'
+                    url: API + url
```

Comment:
Might there should be a check here to make sure `url` is not started with `/` ? 

Though `/api//endpoint` with two slashes won't cause an error, `path.join` or [url.resolve](https://nodejs.org/api/url.html#url_url_resolve_from_to) might be two better alternatives. `(API + url).replace /\/\//, '/'` is ok too, I think.